libraries/common/src/main/java/androidx/media3/common/Format.java

```diff
@@ -160,6 +162,8 @@ public static final class Builder {
     @Nullable private byte[] projectionData;
     private @C.StereoMode int stereoMode;
     @Nullable private ColorInfo colorInfo;
+    private int lumaBitdepth;
```

Comment:
Could you please instead add this bit depth info to the ColorInfo class?

Reply:
I don't have a strong opinion on where this value should go, but I figure both the luma and chroma bitDepth should be in the same class (either ColorInfo or Format)

Reply:
Yes, my apology, the comment was to move both into ColorInfo. Could not attach the comment to multiple lines.

Reply:
Tbh it feels slightly wrong to me seeing luma bitdepth in ColorInfo, and since luma & chroma bitdepth are logically in pair
I would think they should go in Format, however I'm happy to move them to ColorInfo if you guys prefer so.

Reply:
@microkatz final decision ColorInfo or Format?